spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/security/FirebaseAuthenticationAutoConfiguration.java

```diff
@@ -58,13 +58,19 @@
 
 	private static final String ISSUER_TEMPLATE = "https://securetoken.google.com/%s";
 
+	private final String projectId;
+
+	public FirebaseAuthenticationAutoConfiguration(GcpProjectIdProvider gcpProjectIdProvider, FirebaseAuthenticationProperties properties) {
+		this.projectId = (properties.getProjectId() != null) ? properties.getProjectId() : gcpProjectIdProvider.getProjectId();
```

Comment:
I think parenthesis are not needed 😬

Reply:
Indeed! I was just following the same pattern as suggested :)
Removed :)